### ISSUE TITLE
Trying to reuse the model

### ISSUE THREAD

**Issue by Anko59** [FlashTek/attention-is-all-you-need-keras#2]
Hello, I've been trying to train this implementation, but the results are very poor:
While the loss decreases during trainning and the accuracy is exellent, I am not able to have significant results in inference:
![image](https://user-images.githubusercontent.com/25617029/45550643-c00d4600-b82b-11e8-921a-45f3b04d8aaa.png)
![image](https://user-images.githubusercontent.com/25617029/45550762-24c8a080-b82c-11e8-98c5-f5735df6b614.png)
I have tryied with various datasets and parsing methods, these results are with the original model, parser and dataset.

Thank you for your work
